### ISSUE TITLE
Configuration for rule "import/internal-regex" is invalid

### ISSUE THREAD

**Issue by Slapbox** [benmosher/eslint-plugin-import#2101]
An equivalent error also occurs for `import/external-module-folders`.

> Error: .eslintrc:
>        Configuration for rule "import/internal-regex" is invalid:
>        Severity should be one of the following: 0 = off, 1 = warn, 2 = > error (you passed '"^my-namespace"').

**In our `.eslintrc` we have the rule like:**

```js
  "import/internal-regex": "^my-namespace",
```

**and if we try like this we just get a different issue:**

```js
  "import/internal-regex": [1, "^my-namespace"],
```

> 1:1   error    Definition for rule 'import/internal-regex' was not found                                                                                                                                                                                       
> 1:1   error    Definition for rule 'import/external-module-folders' was not found

**Comment by Slapbox** [comment]
Same issue if we use `.eslintrc.js` instead of `.eslintrc`.
